tools/hangar/src/__snapshots__/invalid.test.ts.snap

```diff
@@ -196,13 +192,12 @@ Error at ../../../../examples/tests/invalid/statements_if.w:8:5 | Expected type
 `;
 
 exports[`struct_expansion.w > wing test (--target sim) - invalid > stderr 1`] = `
-"Compilation failed with 6 error(s)
+"Compilation failed with 5 error(s)
 Error at ../../../../examples/tests/invalid/struct_expansion.w:3:15 | \\"bublic\\" is not a field of \\"BucketProps\\"
-Error at ../../../../examples/tests/invalid/struct_expansion.w:4:15 | Expected between 0 and 1 arguments but got 2 when instantiating \\"Bucket\\"
-Error at ../../../../examples/tests/invalid/struct_expansion.w:4:32 | Expected type to be \\"BucketProps?\\", but got \\"num\\" instead
-Error at ../../../../examples/tests/invalid/struct_expansion.w:6:15 | Expected expression
-Error at ../../../../examples/tests/invalid/struct_expansion.w:7:15 | Unknown parser error.
-Error at ../../../../examples/tests/invalid/struct_expansion.w:11:3 | Unknown symbol \\"handler\\""
+Error at ../../../../examples/tests/invalid/struct_expansion.w:7:15 | Expected between 0 and 1 arguments but got 2 when instantiating \\"Bucket\\"
+Error at ../../../../examples/tests/invalid/struct_expansion.w:7:32 | Expected type to be \\"BucketProps?\\", but got \\"num\\" instead
+Error at ../../../../examples/tests/invalid/struct_expansion.w:11:3 | Expected 2 arguments but got 1
+Error at ../../../../examples/tests/invalid/struct_expansion.w:11:15 | Unexpected 'keyword_argument'."
```

Comment:
I don't get this error: `Unexpected 'keyword_argument'."` the previous snapshot made more sense for line 11.

Reply:
ok, that's cool, but is leaky. Maybe we can improve here?

Reply:
https://github.com/winglang/wing/issues/1766